speedtest_cli.py

```diff
@@ -541,6 +541,16 @@ def ctrl_c(signum, frame):
     raise SystemExit('\nCancelling...')
 
 
+def sig_pipe(signum, frame):
+    """Ignore the PIPE signal when trying to output on a closed file handle,
+    and simply exit
+
+    """
+    global shutdown_event
```

Comment:
I've simply copied the global declaration from the `ctrl_c` function, but I don't think the global is useful at all.
This may be a trick for an older python version I've forgotten?